### ISSUE TITLE
Rename the Scroll of Genocide to something else

### ISSUE THREAD

**Issue by webbedspace** [NetHack/NetHack#359]
This choice of word has existed in the game for decades, since Jay Fenlason's Hack 1.0 circa 1985, and gone essentially entirely unquestioned. It was never a good or ethical name, and it should never have been chosen. Genocide is not and never was a distant historical nightmare: it is happening now, throughout the world, in Myanmar, in India, in China, in the United States, in my home of Australia, and many other nations I could list. It is not "merely" systemic and deliberate mass-murder, as this game object depicts, but state-enacted systemic destruction, neglect and suppression of entire schools of culture, science, literature, truth, of everything that makes us human. It is a crime against all humanity, and should never be presented as a tool, or a toy, in any game or narrative setting, fantasy or otherwise.

I'm not bothering to suggest that the scroll's effect itself be removed, as I believe the scroll represents genocide in such a reductionist, sickeningly absurd fashion that it otherwise wouldn't be legible as genocide without the word itself providing a forced preconception - and also because I believe the weight of unthinking historical inertial will act against such a request. I also don't care what alternative name you choose – though I'll remark that since this scroll very specifically affects classes of monsters, and there are many scrolls of the form "<verb> monster", something like "delete class" or "erase class" could come to mind.

I (Leon Arnott) am not new to NetHack or its codebase, and have made very minor contributions throughout the 00's that have merited my name's listing in the Guidebook, so rest assured this is not a request made by a genuine outsider with no visible good faith.

Finally, I would like to pre-emptively respond to the anticipated response that NetHack is a game replete with criminal and disgusting acts presented humorously, juxtaposed with the player's own frequent comical demise. Yes, I am aware of the game's overall tone, but in my eyes if you can't see the vast chasm of weight and importance separating cannibalism, non-consensual sex, animal cruelty, and so forth from the act of genocide, then I deeply question your sincere motivation to developing and stewarding an entertainment product for humans.

**Comment by NullCGT** [comment]
I hadn't really considered this perspective before, and upon reflection, I agree. As taboo as it is to suggest a change to something that's been part of the game for so long, it's very reductionist  to use the name for one of the worst atrocities in history to represent the removal of a letter from the game. More importantly, genocide is not a problem of the past, but something still actively occurring in the world today. The use of "genocide" in such a flippant manner is uncomfortable, and in extremely poor taste.

**Comment by paulwinner** [comment]
I hadn't thought about it this way either. It's a fair point. I suggest
"Scroll of Extinction".

**Comment by ostrosablin** [comment]
+1 for "Scroll of Extinction"

**Comment by copperwater** [comment]
If renamed to "scroll of extinction", what internal changes will be needed in the game to reconcile the new scroll effect ("magical extinction", G_GENOD) from the already-existing concept of extinction ("natural extinction", G_EXTINCT)? Some examples:
* A polymorphed monster or shapeshifter can turn into an extincted monster, but not a genocided one.
* Vampires can become fog clouds if they are extinct, but not genocided.
* The character will merge with their dragon scale mail if the dragon species is extinct, but not if it is genocided.
* Monsters instantly die when growing up into a genocided species, but are okay if it's a merely extinct species.
* Extinctionism is a well-regarded unofficial challenge. Would it become impossible to differentiate between species extincted "the hard way" and "the easy way"?

Not opposed to the rename, just wondering about the reconciliation of the in-game concepts.

**Comment by paulwinner** [comment]
I had forgotten about the existing 'extinction' in the game. I was using
the word as another word for 'genocide'. I didn't intend to imply any
changes, other than the name change, to the code. I'll poke around for
another word to use.

**Comment by paulwinner** [comment]
How about 'annihilation'? The dictionary definitions refer to the something
being obliterated or destroyed. It is used a lot when referring to things
like the results of nuclear bombs, which I think are considered universally
bad and not directed at one particular group in the modern era.

**Comment by k21971** [comment]
You see here a scroll of annihilation - works for me.

**Comment by ostrosablin** [comment]
Yes, perhaps it would be better, as it doesn't conflict with existing concepts and would be a matter of rename.

**Comment by tonehackrl** [comment]
Scroll of Extinction is a great name based on the mechanic, but Extinction already has a specific meaning in the game that is distinct from genocide.

A few of us were brainstorming some other potential names earlier -- I'm partial to Scroll of Removal or Scroll of Banishment, but others discussed were Annihilation, Eviction, Unbecoming, Unmaking, Unbinding, Neverborn, Decimation, Eradication, Extermination, and Elimination.

**Comment by alysdexia** [comment]
[redacted]

**Comment by akolade** [comment]
"Scroll of Annihilation" sounds good to me, unless there is some similar existing spell in other fantasy works that could be borrowed.

**Comment by Rhialto** [comment]
Decimation: killing every tenth, so not accurate.
Extermination: could also be called the Dalek Scroll.

**Comment by paulwinner** [comment]
I would like to add something that will only make sense to some of us. When
I first read the initial email I had an immediate thought and felt that if
I shared it I would seem silly. I can't get it out of my mind, so here it
is. I think that Izchak would be behind this effort. This is one of those
things that he put his foot down when it was brought up. I am reminded of
his response to removing a fortune that mentioned gypsies.

**Comment by paulwinner** [comment]
On the "annihilation" vs "extermination" discussion.

Doing some on-line thesaurus, dictionary, etc., browsing. I think
'annihilation' wins out over 'extermination'

I didn't read any references to something like genocide, Nazis, etc. on
"annihilation" searches.
an act or instance of annihilating, or of completely destroying or
defeating someone or something: the brutal *annihilation* of millions of
people. the state of being annihilated; total destruction; extinction: fear
of nuclear *annihilation*. Physics.

"Extermination" on the other hand came up with many Holocaust links in the
top search hits.
*Names of the Holocaust* vary based on context. "The Holocaust
<https://en.wikipedia.org/wiki/The_Holocaust>" is the name commonly applied
in English since the mid-1940s to the systematic extermination of
(Wikipedia) for example.

**Comment by ostrosablin** [comment]
It just occured to me, that "annihilation" as in "scroll of annihilation" lacks context to hint what exactly it's annihilating. Original "genocide" suggests that it will do something to living things, that is, monsters. "Annihilation" on other side, unless you know the context beforehand, could be related to pretty much anything - map features, items, monsters. It doesn't suggest that it will annihilate particularly monsters and isn't immediately obvious unless you're spoiled. Guess such name could be confusing for both new players and NetHack veterans.

Name should directly suggest that it's killing/eliminating monsters, not some abstract objects.

Perhaps, it should be called "Scroll of Annihilate Monster", which follows the pattern of "Scroll of Create Monster" (and would be it's counterpart, in a way).

Another option I see is "Scroll of Eliminate Monster" - that sounds even more neutral, because elimination doesn't strictly assume mass murder (could be interpreted as, e.g. teleporting away all entities of the selected species out of Mazes of Menace, though for this interpretation to work it would require changing current genocide mechanics not to leave corpses of eliminated monsters, but this would prevent drops from monsters, so I'm not very fond of this idea).

**Comment by todb-r7** [comment]
I really like the complimentary "Create Monster" versus "Annihilate Monster."

I might suggest Create/Destroy Monster? After all, the cursed version ends up acting like a targeted Create Monster anyway. 

You could argue that "Destroy Monster" might not carry the intended gravitas of "Genocide" or "Annihilate" but "Destroy" is shorter and easier to spell, and newbies who read it and see the resulting message of "Wiped out all newts" will get the idea pretty quickly what it's intended to do.

**Comment by copperwater** [comment]
I don't like "Destroy Monster" - the connotation I get from it is that it destroys _one_ individual monster, and maybe a blessed one destroys multiple monsters. Actually, this is a problem with "Annihilate Monster" too. Sounds like it's targeted at one monster, or maybe at most does something like UnNetHack's genocide scroll which wipes out all monsters of a given species on the current level only (unless blessed). _Maybe_ it could be something like "scroll of annihilate species", but that sounds very clunky to me.

I find that I'm still favoring a single-word approach, with "extinction" or perhaps "eradication".

**Comment by todb-r7** [comment]
Yeah I'm not totally sold on "Destroy Monster" either. "Scroll of Extinction" sounds pretty good to me.

**Comment by akolade** [comment]
The only problem with "Scroll of Extinction" is that "extinction" is already a concept in Nethack.
https://nethackwiki.com/wiki/Extinctionism

Otherwise it'd be perfect.

**Comment by todb-r7** [comment]
Oh heck, @akolade, totally right, and discussed above. Rats.

**Comment by offalynne** [comment]
Annihilation makes for good acrophony in that the result removes a letter. I agree that most of the alternatives posed do not necessarily communicate “remove all” very effectively, but as the original post details, the existing term does not shorthand that result very well either. It’s unfortunate extinction is unavailable as a keyword, perhaps exterminate (Scroll of Extermination or Scroll of Exterminate Monster) is the best proposal for this reason.

**Comment by latogato** [comment]
I have only somewhat funny ideas:
Scroll of ban monster
Scroll of restraining order

**Comment by latogato** [comment]
Scroll of Obliteration.

Obliteration hints you can't polymorph into the obliterated monster anymore, while extermination and annihilation don't.

**Comment by Scott-TT** [comment]
The entire "functional point" of the (uncursed) scroll is to kill a massive population, specifically selected/targetted because of their race/ethnics/genes. This is essentially the very definition of the word "genocide".

The social, political, or even moral considerations one might want to attach to it (even with the best reasons) doesn't change the cold, objective, truth that this word is the proper description for the scroll's effect. 

Whatever connotations one might attach (or not) to the term, with good reasons (or not) should have no bearing on this. As a matter of fact the genocide "ingame" is at least just as malicious, thorough, unescapable and definitive than any other. Caring about the victims is another matter entirely.

**Comment by theTrueMikeBrown** [comment]
I think that extinction is the right word. I know that it has pre-existing meaning in the game, but really, adding on an adjective to describe the type of extinction seems to be a simple way to deal with this.

Magical Extinction is the hard "you can never see this monster again" version, and Mundane Extinction is the "this will never be generated randomly again, but may exist through other means"

**Comment by Scott-TT** [comment]
Extinction lacks target-specificity, it could apply to "any female", "monsters on dlvl 24", "monsters with hands", "monsters named Bob", "candles", "light radius", ... 

On top of this, the very point of the renaming seems to be "the action associated to this word is bad, so we shouldn't use the word". Seems like ostrich strategy as it won't change anything or help anyone. On the other hand, some may like to RP more than "whack-a-pixel-doo" (i do), and genocide IS very much everything described in OP's first paragraph as basis for how this differs from that. Nobody will miss the L, but have you thought about all the dwarves you monsters ?

**Comment by jeremyn** [comment]
Another problem with the word "genocide" is that genocide in the real world applies by definition to people whereas in NetHack it's also applied to, for example, ants, which trivializes actual genocide.

I agree with where @latogato is going in https://github.com/NetHack/NetHack/issues/359#issuecomment-700583889. As the player I don't care about killing anything with "genocide", I just want the targets out of my way for the duration of the game. For all it matters, I can imagine that the rest of the dungeon is crammed full of them, and stage crew are hustling them out of the way as I move around. "Annihilation" or "extinction" are unnecessarily harsh.

So I'd go with "banish monster." Another advantage of this is that the cursed version effectively "summons" your target, which is the natural opposite of the word "banish." Bringing in 4-6 creatures isn't really the opposite of "annihilating" an entire species. In any case the name of the scroll isn't going to capture exactly what it does any more than "fire" or "punishment" do. There's no perfect name, just, obviously, less bad ones.

(There's a whole other discussion to be had about whether it's okay to refer to an entire sentient species as "monsters", but that's for another time.)

**Comment by bryceco** [comment]
I think @Scott-TT nailed it pretty well. It's abhorrent in current society, and it's equally abhorrent in the NH universe. When you perform that act you should be aware that it's an unconscionable act. But that doesn't mean we should rename it.

**Comment by kristjanlink** [comment]
+1 for Scroll of Banishment, it makes the most sense

**Comment by jeremyn** [comment]
It's unethical to have a game where a player can explicitly commit genocide without very serious penalties and without exploring the consequences. It would be extremely unlikely -- basically impossible -- that NetHack could do this appropriately. People who are interested in that sort of game can play something like "Undertale" or "This War of Mine."

In checkers, when one piece jumps another and removes it from the board, that's calling "capturing," which is a neutral term. Imagine for historical reasons it was called something more extreme like "murdering" and that today people argue to change it to something else. Would other people disagree, claiming that changing the name would be "inauthentic"? No, or at least, I hope not. The playing pieces are so stylized that the name we give to the action defines what is happening.

I'm unclear how GitHub issues translate into actual changes in the game for big things like this -- big in terms of changing a longstanding aspect of the game, not in terms of code changes, which I think would be relatively minimal. Are the developers actually interested in this? Would be a PR changing the name of the action be seriously considered?

**Comment by psymar** [comment]
> In checkers, when one piece jumps another and removes it from the board, that's calling "capturing," which is a neutral term. Imagine for historical reasons it was called something more extreme like "murdering" and that today people argue to change it to something else. 

I have absolutely heard people say they "killed" a rook/knight/whatever in chess, although "capturing" is the official term.  (of note:  "checkmate" or "shah mat" does not mean "the king is dead" but "the king is helpless".)

Shiren already uses "scroll of removal"; could make it "scroll of species removal" if we want to make it more specific, but then you run into things like being able to genocide player roles (which are not species).

**Comment by NullCGT** [comment]
In SpliceHack, we changed the name to Scroll of Annihilation, and it's worked fairly well so far.

> Shiren already uses "scroll of removal"; could make it "scroll of species removal" if we want to make it more specific, but then you run into things like being able to genocide player roles (which are not species).

I believe that some translations of Shiren use "Scroll of Sealing" as an alternative to "Scroll of Removal," but sealing implies a completely different concept.* Personally, I'm a big fan of Scroll of Removal. It's short, succinct, and gets across what the scroll does.

\* There's also an old fan translation of Shiren 1 for Famicom which refers to it as a Scroll of Genocide, supposedly as a reference to Rogue/Hack/NetHack. I bring this up for two reasons: First, because I think it's an interesting and obscure fact. Second, I think it's a good example of how the flippant usage of the word in NetHack can directly influence the topic's treatment in other mediums, and why changing it is important.

**Comment by Kufat** [comment]
"Annihilation" also lends itself to some wordplay. "Sent in some foo" for cursed scroll monster generation has always seemed a bit drab, but with "annihilation" you could have "Foo annihilated!" for the normal case and "Foo ex nihilo!" when monsters are generated.

**Comment by Nasrudith** [comment]
> I think @Scott-TT nailed it pretty well. It's abhorrent in current society, and it's equally abhorrent in the NH universe. When you perform that act you should be aware that it's an unconscionable act. But that doesn't mean we should rename it.

While I'm dubious of any supposed moralistic grounds for changes there may still be some small improvements for other semantic changes. Even though I would miss phrases like "Quit while still on Charon's boat."

Genocide is majorly underrated in alignment though. It also only affects your moral record by +/- 1 for human species genocided if lawful or chaotic (although only the Keystone Kops qualify).  Even if genocide was explicitly stated only apply to the dungeon itself that is still massively underrated. 

Genocide also has some other inconsistency issues like Pacifism not being broken by it. There could be some merit to making it simply "Banishment' or "Exile". Displacing an entire species from their homes in the dungeon would be a very rude disruption but if they were relocated to a viable alternative otherworldly environment the +1/-1 would fit better as they are all merely upset instead of killed or injured. Sending in some <foo> as a backfire for a scroll of banishment would also fit as a reversal in both senses of "failed to send them away from you" and "sent them away from someplace else". Minor side effects like auto-banishment being treated as an escape would fit in that light.

**Comment by BigMeatyClaws** [comment]
Why is everyone obsessed with trying install some societal justice in a game about @ killing a lot of things?

How about this: do not rename anything. There is no need to. The only victims of genocide in nethack are virtual characters with no feelings or souls. Real victims of genocide do not play nethack, I assure you. And if they do, I request that they reply to me so I can eat my hat.

Please stop getting angry about immensely insignificant details in every single aspect of your life on behalf of people who simply do not care much about such details.

**Comment by jeremyn** [comment]
I think at this point this issue is waiting on the DevTeam to show interest and guide the discussion. Though this is a relatively small change technically, even if we add things we haven't yet discussed such as permanently aliasing "annihilation" (or whatever) with "genocide" when wishing to avoid breaking countless wishing guides, unfortunately "genocide" is an iconic name, so we can't just sneak the change in like a YAFM. The change will need enthusiastic DevTeam support.

In the meantime, some of the folks here might find this article interesting: "[Cardboard Genocide. Board Game Design as a Tool in Holocaust Education](https://www.gamejournal.it/cardboard-genocide/)". It provides some background and a case study on game playing and game design as tools for understanding the Holocaust. However, beyond the article itself, it's interesting to consider that all of the references at the end -- books, games, films -- are dated after 1985 when @webbedspace says "genocide" was added to Hack 1.0. NetHack is very niche and old-school now but of course 20-30 years ago it was relatively huge. A lot of today's game designers don't play it but the games they do play were designed by people who did, or the people who designed their games did, and so on. Billions of people have played games where, somewhere in the game design "family tree", some designer at least momentarily thought of genocide as a way to "wipe out all L". I'm not sure what that fact means in the bigger picture, exactly, other than "nothing good".

**Comment by nhmall** [comment]
Moved to [Feature Requests](https://github.com/NetHack/NetHack/wiki/Feature-Requests) page, with a link back to this Issue and its comments.

Closing the issue.
